### PR TITLE
seventh task

### PR DESCRIPTION
**Pull request by YANGMAO-ZHANG**
@Felixnoo

### CODE REVIEW

**Comment by YANGMAO-ZHANG** [comment]
#1854

File: content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md [hunk 1]
```diff
@@ -1,67 +1,74 @@
 ---
-title: "Use Nexus in Pipelines"
-keywords: 'KubeSphere, Kubernetes, Pipeline, Nexus, Jenkins'
-description: 'Learn how to use Nexus in pipelines on KubeSphere.'
-linkTitle: "Use Nexus in Pipelines"
+title: "在流水线中使用 Nexus"
+keywords: 'KubeSphere, Kubernetes, 流水线, Nexus, Jenkins'
+description: 'Learn how to use Nexus in pipelines on KubeSphere.学习如何在 KubeSphere 流水线上使用 Nexus'
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md:4]: `Learn how to use Nexus in pipelines on KubeSphere.学习如何在 KubeSphere 流水线上使用 Nexus`
->
`学习如何在 KubeSphere 流水线中使用 Nexus。`

File: content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md [hunk 2]
```diff
@@ -1,67 +1,74 @@
...
+linkTitle: "在流水线中使用 Nexus"
 weight: 11450
+
+
 ---
 
-[Nexus](https://www.sonatype.com/products/repository-oss) is a repository manager that stores, organizes, and distributes artifacts. With Nexus, developers can have better control over the artifacts needed in a development process.
+[Nexus](https://www.sonatype.com/products/repository-oss) 是存储、组织和分发工件的存储管理器。使用 Nexus 的开发者可以更好的控制开发过程中所需的工件。
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md:11]: `工件` -> `制品`
Matter of style.

File: content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md [hunk 3]
```diff
@@ -1,67 +1,74 @@
...
+
+本教程概述如何在 KubeSphere 流水线上使用 Nexus。
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md:13]: `概述` -> `演示`
`上` -> `中`

File: content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md [hunk 4]
```diff
@@ -1,67 +1,74 @@
...
+- [启用 KuberSphere DevOps 系统](../../../../docs/pluggable-components/devops/)。
 
-## Prerequisites
+- [准备 Nexus 实例](https://help.sonatype.com/repomanager3/installation)。
 
-- You need to [enable the KubeSphere DevOps System](../../../../docs/pluggable-components/devops/).
-- You need to [prepare a Nexus instance](https://help.sonatype.com/repomanager3/installation).
-- You need to have a [GitHub](https://github.com/) account.
-- You need to create a workspace, a DevOps project (for example, `demo-devops`), and an account (for example, `project-regular`). This account needs to be invited into the DevOps project with the role of `operator`. For more information, see [Create Workspaces, Projects, Accounts and Roles](../../../quick-start/create-workspace-and-project/).
+- [GitHub](https://github.com/) 帐户。
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md:21]: Add `准备一个` at the beginning of this sentence.

File: content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md [hunk 5]
```diff
@@ -1,67 +1,74 @@
...
 
-## Hands-on Lab
+- 创建一个工作空间，一个 DevOps 工程（例如，`demo-devops`）和一个帐户（例如，`project-regular`）。`project-regular` 需要被邀请至 DevOps 工程中并赋予 `operator` 角色。有关更多信息，请参考[创建企业空间、项目、帐户和角色](../../../quick-start/create-workspace-and-project/)。
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md:23]: `工作空间，` -> `企业空间、`

File: content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md [hunk 6]
```diff
@@ -1,67 +1,74 @@
...
+2. 转到**仓库**页面，您可以看到 Nexus 提供了三种仓库类型。
 
-   ![repo-type](/images/docs/devops-user-guide/examples/use-nexus-in-pipeline/repo-type.png)
+   - `proxy`：远程仓库代理，用于下载资源并将其作为缓存存储在 Nexus 上。
 
-3. You can click a repository to view its details. For example, click **maven-public** to go to its detail page and you can see its URL.
+   - `hosted`：仓库在 Nexus 存储工件。
 
-   ![maven-public-url](/images/docs/devops-user-guide/examples/use-nexus-in-pipeline/maven-public-url.png)
+   - `group`：一组配置了 Nexus 的仓库。
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md:37]: `一组配置了 Nexus 的仓库` -> `一组已配置好的 Nexus 仓库`

File: content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md [hunk 7]
```diff
@@ -1,67 +1,74 @@
...
+   ![maven-public-url](/images/docs/zh-cn/devops-user-guide/examples/use-nexus-in-pipeline/maven-public-url.png)
 
-   ![click-pom](/images/docs/devops-user-guide/examples/use-nexus-in-pipeline/click-pom.png)
+### 步骤 2：在 GitHub 仓库修改 `pom.xml`
 
-3. Click <img src="/images/docs/devops-user-guide/examples/use-nexus-in-pipeline/github-edit-icon.png" height="18px" /> to modify the code segment of `<distributionManagement>` in the file. You need to keep the value of `<id>` the same as the value in the ConfigMap modified in step 2 and use the URLs of your own Nexus repositories . 
+1. 登录 GitHub，fork [示例仓库](https://github.com/devops-ws/learn-pipeline-java)到您的 GitHub 帐户。
 
-   ![modify-pom](/images/docs/devops-user-guide/examples/use-nexus-in-pipeline/modify-pom.png)
+2. 在您的 **learn-pipline-java** GitHub 仓库中，点击 root 文件下的文件 `pom.xmnl`。
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md:49]: pom.xmnl -> pom.xml

File: content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md [hunk 8]
```diff
@@ -1,67 +1,74 @@
...
 
-4. When you finish, click **Commit changes** at the bottom of the page.
+   ![click-pom](/images/docs/zh-cn/devops-user-guide/examples/use-nexus-in-pipeline/click-pom.png)
 
-### Step 3: Modify the ConfigMap
+3. 在文件中点击 <img src="/images/docs/zh-cn/devops-user-guide/examples/use-nexus-in-pipeline/github-edit-icon.png" height="18px" /> 以修改 `<distributionManagement>` 代码片。设置 `<id>` 并使用您的 Nexus 仓库的 URL。
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md:53]: `代码片` -> `代码片段`

File: content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md [hunk 9]
```diff
@@ -1,67 +1,74 @@
...
+   ![modify-pom](/images/docs/zh-cn/devops-user-guide/examples/use-nexus-in-pipeline/modify-pom.png)
 
-2. Select **ConfigMaps** under **Configurations**. On the **ConfigMaps** page, select `kubesphere-devops-system` from the drop-down list and click `ks-devops-agent`.
+4. 当您完成以上步骤，点击页面下方的 **Commit changes**。
 
-   ![ks-devops-agent](/images/docs/devops-user-guide/examples/use-nexus-in-pipeline/ks-devops-agent.png)
+### 步骤 3：修改 ConfigMap
 
-3. On the detail page, click **Edit YAML** from the **More** drop-down menu.
+1. 使用 `admin` 帐户登录 KubeSphere web 控制台，点击左上角的**平台**，选择**集群管理**。
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md:61]: `平台` -> `平台管理`

File: content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md [hunk 10]
```diff
@@ -73,15 +80,15 @@ This tutorial demonstrates how to use Nexus in pipelines on KubeSphere.
...
 
    {{< notice note >}}
 
-   `<id>` is a unique identifier you set for your Nexus. `<username>` is your Nexus username. `<password>` is your Nexus password. You can also configure a `NuGet API Key` on Nexus and use it here for better security.
+   `<id>` 是您在步骤 2 设置给 Nexus 的唯一标识符。 `<username>` 是您的 Nexus 用户名。 `<password>` 是您的 Nexus 的密码。您也可以在 Nexus 上面配置  `NuGet API Key`，以获得更高的安全性。
 
    {{</ notice >}}
 
-5. Continue to find the code segment of `<mirrors>` and enter the following code:
+5. 继续找到 `<mirrors>` 代码分片，然后输入一下代码：
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md:91]: `代码分片` -> `代码片段`

File: content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md [hunk 11]
```diff
@@ -94,31 +101,31 @@ This tutorial demonstrates how to use Nexus in pipelines on KubeSphere.
    </mirrors>
    ```
 
-   ![enter-mirror-code](/images/docs/devops-user-guide/examples/use-nexus-in-pipeline/enter-mirror-code.png)
+   ![enter-mirror-code](/images/docs/zh-cn/devops-user-guide/examples/use-nexus-in-pipeline/enter-mirror-code.png)
 
    {{< notice note >}}
 
-   `<id>` is a unique identifier you set for your Nexus. `<name>` is the Nexus repository name. `<url>` is the URL of your Nexus repository. `<mirrorOf>` is the Maven repository to be mirrored. In this tutorial, enter `*` to mirror all Maven repositories. For more information, refer to [Using Mirrors for Repositories](http://maven.apache.org/guides/mini/guide-mirror-settings.html).
+   `<id>` 是您在步骤 2 设置给 Nexus 唯一标识符。 `<name>` 是 Nexus 仓库的名字。 `<url>` 是您 Nexus 仓库的 URL。 `<mirrorOf>` 是 要镜像的 Maven 仓库。在本教程，输入 `*` 制作所有 Maven 仓库的镜像。有关更多信息请参考[为仓库使用镜像](http://maven.apache.org/guides/mini/guide-mirror-settings.html)。
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md:108]: Delete the space between `是` and `要镜像`

Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md:108]: `制作所有 Maven 仓库的镜像` -> `镜像所有 Maven 仓库`

File: content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md [hunk 12]
```diff
@@ -94,31 +101,31 @@ This tutorial demonstrates how to use Nexus in pipelines on KubeSphere.
...
    {{</ notice >}}
 
-6. When you finish, click **Update**.
+6. 当您完成，点击**更新**。
 
-### Step 4: Create a pipeline
+### 步骤 4：创建流水线
 
-1. Log out of the KubeSphere web console and log back in as `project-regular`. Go to your DevOps project and click **Create** on the **Pipelines** page.
+1. 登出 KubeSphere web 控制台，使用帐户 `project-regular` 登录。转到 DevOps 工程，然后在**流水线**页面点击**创建**。
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md:116]: `web` -> `Web`

File: content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md [hunk 13]
```diff
@@ -94,31 +101,31 @@ This tutorial demonstrates how to use Nexus in pipelines on KubeSphere.
...
+4. 点击流水线进入它的详情页面，然后点击**编辑 Jenkinsfile**。
 
-   ![click-edit-jenkinsfile](/images/docs/devops-user-guide/examples/use-nexus-in-pipeline/click-edit-jenkinsfile.png)
+   ![click-edit-jenkinsfile](/images/docs/zh-cn/devops-user-guide/examples/use-nexus-in-pipeline/click-edit-jenkinsfile.png)
 
-5. In the displayed dialog box, enter the Jenkinsfile as follows. When you finish, click **OK**.
+5. 在出现的会话窗口中，在 Jenkinsfile 中输入如下内容。当您完成，点击**确定**。
 
    ```groovy
    pipeline {
```
Comment (Felixnoo) [inline content/zh/docs/devops-user-guide/examples/use-nexus-in-pipelines.md:131]: `https://github.com/Felixnoo/learn-pipeline-java.git` 
->
`https://github.com/YANGMAO-ZHANG/learn-pipeline-java.git`

**Review by Felixnoo — CHANGES_REQUESTED** [review]
Please take a look at the comments, thanks.

**Comment by Felixnoo** [comment]
/assign @Patrick-LuoYu 
Hi, this PR is submitted by the OSSC student. I reviewed it already. Please take a look at it when you are available, thanks.

**Pull request merged**